tests/Tests/Fixture/FixtureManager.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace OpenEMR\Tests\Fixture;
+
+require_once("library/sql.inc");
+
+/**
+ * Provides OpenEMR Fixtures/Sample Records to test cases as Objects or Database Records
+ * 
+ * The FixtureManager generates sample records from JSON files located within the Fixture namespace.
```

Comment:
In regards to this guidance, instruction I think it may be easier initially to add methods to the class rather than making an inheritance hierarchy out of it. The latter approach seems that it would be complex for what we're trying to accomplish initially.